db/migrate/20220221223739_add_similarity_to_name_search.rb

```diff
@@ -1,7 +1,8 @@
 class AddSimilarityToNameSearch < ActiveRecord::Migration
   def change
     enable_extension :pg_trgm
+    enable_extension :btree_gin
 
-    add_index :supporters, :name, name: :name_search_idx, order: {name: :gin_trgm_ops}
+    add_index :supporters, :name, name: :name_search_idx, using: :gin, order: {name: :gin_trgm_ops}
```

Comment:
I suggest this be changed to:

```ruby
class AddSimilarityToNameSearch < ActiveRecord::Migration
  def up
    enable_extension :pg_trgm
    enable_extension :btree_gin
    execute <<-SQL
      CREATE INDEX name_search_idx
      ON supporters
      USING gin (name gin_trgm_ops);
    SQL
  end

  def down
    remove_index :supporters, name: :name_search_idx
  end
end
```

Reply:
I did a few tests with that and so far it didn't work not using the  `%` operator to compare the similarity, the current tests failed, but I'll test more.